Build fails on macOS 10.13 ssdeep bottles are not correct arch

<!-- Thank you for contributing to osquery! -->

# Bug report

<!--
Please follow this template.
Before submitting an issue search for duplicates.
-->

### What operating system and version are you using?

macOS 10.13.6 (17G65)


### What version of osquery are you using?
commit cf59b05

### What steps did you take to reproduce the issue?

1. `git clone` the osquery repository
1. run `make depsclean` to ensure you do not have any existing osquery dependencies installed
1. run `make deps`
1. run `make`

### What did you expect to see?

A successful build

### What did you see instead?

```
[ 81%] Linking CXX executable osquery_additional_tests
ld: warning: ignoring file /usr/local/osquery/lib/libfuzzy.a, file was built for archive which is not the architecture being linked (x86_64): /usr/local/osquery/lib/libfuzzy.a
Undefined symbols for architecture x86_64:
  "_fuzzy_hash_filename", referenced from:
      osquery::tables::genHashForFile(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, osquery::QueryContext&, std::__1::vector<std::__1::map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::less<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > >, std::__1::allocator<std::__1::map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::less<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > > > >&) in lto.o
  "_fuzzy_compare", referenced from:
      osquery::sqliteSsdeepCompareFunc(sqlite3_context*, int, sqlite3_value**) (.llvm.D284339E) in lto.o
ld: symbol(s) not found for architecture x86_64
```

The commit that prevents me from building osquery is b7079f41f55f61804f7b9f088db570b73aef536. If I roll back before this commit and follow the repro steps, I get a successful build.

--- a/tools/provision/formula/ssdeep-cpp.rb
+++ b/tools/provision/formula/ssdeep-cpp.rb
@@ -6,13 +6,13 @@ class SsdeepCpp < AbstractOsqueryFormula
   license "GPL-2.0+"
   url "https://github.com/ssdeep-project/ssdeep/releases/download/release-2.14.1/ssdeep-2.14.1.tar.gz"
   sha256 "ff2eabc78106f009b4fb2def2d76fb0ca9e12acf624cbbfad9b3eb390d931313"
-  revision 200
+  revision 201
 
   bottle do
     root_url "https://osquery-packages.s3.amazonaws.com/bottles"
     cellar :any_skip_relocation
-    sha256 "00ccb3820f19ca73e5cf553da5623d118843fad1516d5bf283eae403ae55298d" => :x86_64_linux
-    sha256 "64a6e3794335f0b24813ad3b22d40ed2bb32ad8040414e59022dfeb148e0c0e1" => :sierra
+    sha256 "87b379ad5ac57c463644b93212a110ebd10ae6363d5a58b1b81d623ac5394cb0" => :sierra
+    sha256 "adf265c3b2c3a260c48be6fd8a5bedc0a99b39e836e874abe71373c84ddd693b" => :x86_64_linux
   end
 
   def install

